src/send/SendEnterAmount.tsx

```diff
@@ -322,7 +322,7 @@ function SendEnterAmount({ route }: Props) {
                 <>
                   <FastImage source={{ uri: token.imageUrl }} style={styles.tokenImage} />
                   <Text style={styles.tokenName}>{token.symbol}</Text>
-                  <DownArrowIcon color={Colors.gray5} />
+                  {!forceTokenId && <DownArrowIcon color={Colors.gray5} />}
```

Comment:
Seems like this is still touchable, which allows changing the token. I don't think we should allow changing tokens if forceTokenId is set.
Also, can we add unit tests

Reply:
okay I can update to remove the ability change, the issue just said to remove the icon so wasn't sure if we should also remove the ability to change, but should be easy to update.

Reply:
@finnian0826 Nitya also said in a comment that the user should not have the ability to change tokens
https://linear.app/valora/issue/ACT-963/handle-forcetokenid-on-enter-amount-screen#comment-4ab5f930